### ISSUE TITLE
Can we replace Jaeger's domain model with OTel proto?

### ISSUE THREAD

**Issue by objectiser** [jaegertracing/jaeger-opentelemetry-collector#30]
Related to https://github.com/jaegertracing/jaeger-opentelemetry-collector/issues/24#issuecomment-592631487

**Comment by pavolloffay** [comment]
This should be open in the jaegertracing/jaeger repo. In this project we just consume components from the jaeger core.

**Comment by pavolloffay** [comment]
I would consider removing this from the acceptance criteria met milestone. How the internal code works should not block the acceptance criteria for in-place replacement of Jeger collector with Jaeger otel collector.

**Comment by objectiser** [comment]
Fair point - lets see if we can do a quick review of the milestone issues on todays call to identify the important ones.
